index.js

```diff
@@ -7,6 +7,19 @@ module.exports.parse = async ({ message, defaultSchemaFormat }) => {
   message['x-parser-original-payload'] = message.payload;
   message.payload = transformed;
   delete message.schemaFormat;
+
+  async function handleProtocolKey() {
+    for (const protocol of Object.keys(message.bindings)) {
```

Comment:
I think we can go straight to Kafka protocol instead of iterating. I don't know any other protocol that has a `key` property that can be defined with Avro.

Reply:
if fact this was my first approach but i guess maybe new EDI technologies in future can handle message key like `nats.io` ... but ok i will change this implementation to only support the kafka protocol

Reply:
done @fmvilas